Makefile

```diff
@@ -25,29 +25,31 @@ help: ## This help.
 build: ui build-all-go ## Build All
 
 .PHONY: build-all-go
-build-all-go: backend cli ## Build All GO executables
+build-all-go: bin/vmclarity-apiserver bin/vmclarity-cli bin/vmclarity-orchestrator bin/vmclarity-ui-backend ## Build All GO executables
 
 .PHONY: ui
 ui: ## Build UI
 	@(echo "Building UI ..." )
 	@(cd ui; npm i ; npm run build; )
 	@ls -l ui/build
 
-.PHONY: backend
-backend: ## Build Backend
-	@(echo "Building Backend ..." )
-	go build -race -o bin/vmclarity-backend cmd/vmclarity-backend/main.go
+bin/vmclarity-orchestrator: $(shell find api) $(shell find cmd/vmclarity-orchestrator) $(shell find pkg) go.mod go.sum ## Build vmclarity-orchestrator
```

Comment:
`$(shell find api) $(shell find cmd/vmclarity-orchestrator) $(shell find pkg) go.mod go.sum` - I'm not familiar with that, what does it do?

Reply:
This is the traditional usage for `make`. It tells make that all these files are bin/vmclarity-orchestrator's dependency, and if any of those files have a newer timestamp than bin/vmclarity-orchestrator then it needs to rebuild it. If none of those files have been updated then bin/vmclarity-orchestrator won't get rebuilt, and it will output that there is nothing to do.